main.go

```diff
@@ -236,6 +275,31 @@ func main() {
 		os.Exit(1) //nolint:gocritic
 	}
 
+	// create disruption rollout reconciler
+	disruptionRolloutReconciler := &controllers.DisruptionRolloutReconciler{
+		Client:  mgr.GetClient(),
+		BaseLog: logger,
+		Scheme:  mgr.GetScheme(),
+	}
+
+	if err := disruptionRolloutReconciler.SetupWithManager(mgr); err != nil {
```

Comment:
It would be really great if we could gate turning on the rollout (and cron!) controllers based on a config value. That way we can safely release the controller even as we continue to work on these